### PR TITLE
Draft of EQ1 checkit

### PR DESCRIPTION
**Pull request by siwelwerd**
Addressing https://github.com/TeamBasedInquiryLearning/precalculus/issues/393

In Task 2, they are asked to write the solution in interval notation and represent it on a number line, as in [Activity 1.1.6(d)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EQ1.html#EQ1-3-7-5).  These number lines are produced in the book as tikz image.  @StevenClontz I need help with producing that kind of image in the solution to Task 2 in Checkit.

### CODE REVIEW

**Comment by siwelwerd** [comment]
This sageplot doesn't look so bad?

image


```
p=line( [(3,0), (10,0)], color='blue', thickness=5,zorder=10)
p+=arrow((-10.5,0),(10.5,0),ticks=[1,[]],color='black',zorder=10,head=2)
p+=text("(",(3,0), color='blue', fontweight=1000,zorder=10,horizontal_alignment='center',fontsize='xx-large')
p.axes_color('white') 
p.show()
```

**Comment by siwelwerd** [comment]
A bit more tweaking

image

```
p=line( [(3,0), (10,0)], color='blue', thickness=8,zorder=10,alpha=0.7)
p+=arrow((-10.5,0),(10.5,0),ticks=[1,[]],color='black',zorder=10,head=2)
p+=text("[",(3,0), color='blue', fontweight=1000,zorder=10,horizontal_alignment='center',fontsize=30,alpha=0.7)
p.axes_color('white') 
p.show()
```

@AbbyANoble  What do you think?  Is this a reasonable number line for the interval $[3,\infty)$ for a Checkit solution?

**Comment by AbbyANoble** [comment]
Looks good! Except why is 0 missing?

**Comment by siwelwerd** [comment]
It's really a 2d plot but with the y axis covered up in white, and Sage doesn't seem to like to label the origin.  Not sure how to fix that.

**Comment by siwelwerd** [comment]
@StevenClontz Any ideas on getting the zero? I tried hacking in adding a zero with text but it looks bad.

**Comment by siwelwerd** [comment]
We cant use the `show` though, so how else can we do `axes=False`?

**Comment by StevenClontz** [comment]
```python
P = arrow((0,0),(10,0),color="black", width=1, arrowsize=1)
P += arrow((0,0),(-10,0),color="black", width=1, arrowsize=1)
for i in range(-9,10):
    P += line([(i,-0.2),(i,0.2)],color="black")
    P += text(f"${i}$", (i,-0.6),color="black")
P += arrow((3.3,0),(10,0),color="#0088ff", width=3, arrowsize=3)
P += circle((3.3,0),0.2,color="#0088ff",fill=True)
P += circle((3.3,0),0.15,color="white",fill=True)
P.axes(False)
P
```

https://sagecell.sagemath.org/?z=eJyVj7EKwjAQhneh7xBqhwTTkhiUKmR17uAmDrEm9jC0kFZaFN_d2qJUi4O3HAf3ffdfgiRSzhU1xowyQjHvWlrYwkn_YFV69imq4VhlktN-tYSrlpx4kwTNvujwH9wUDgGCHDmVnzQOV5QzsvYmqK1ObSHXeIeBhiyat3agz77_1JMBUOmmwsYPbnAP2rs9uSQjYBhcRGL8-JSxODbmnV0Ms4uXIQWXWv1WtOFGvAFr5dZd9E-GL15QnUGlv5BINbrEG2XLbnwANxl24w==&lang=sage&interacts=eJyLjgUAARUAuQ==

image

**Comment by StevenClontz** [comment]
I'll implement now

**Comment by AbbyANoble** [comment]
The issue I'm seeing here is that it's an open dot rather than a parenthesis. Our book uses the parentheses/bracket notation on the lines. And it also looks like the interval (3.1-ish, infinity) not (3,infinity).

**Comment by siwelwerd** [comment]
If you like this we can update the book with this

**Comment by AbbyANoble** [comment]
I like the parentheses/bracket notation a little better because it helps them write the interval notation. We used that in multiple sections. @tdegeorge, @kathypinzon -- What do y'all think?

**Comment by StevenClontz** [comment]
image

**Comment by StevenClontz** [comment]
swapping out dots for brackets isn't too hard

**Comment by AbbyANoble** [comment]
That looks good for now. We can see what Tonya and Kathy prefer!

**Comment by StevenClontz** [comment]
image

**Review by AbbyANoble — APPROVED** [review]
looks great!

File: precalculus/exercises/outcomes/EQ/EQ1/generator.sage [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        if ineq=='>':
+            if a*b-d>0:
+                interval = f"({solution_ineq},\\infty)"
+            if a*b-d<0:
+                interval = f"(-\\infty, {solution_ineq})"        
+        if ineq=='>=':
+            if a*b-d>0:
+                interval = f"[{solution_ineq},\\infty)"
+            if a*b-d<0:
+                interval = f"(-\\infty, {solution_ineq}]"
```
Comment (siwelwerd) [inline precalculus/exercises/outcomes/EQ/EQ1/generator.sage:56]:
```suggestion
        if ineq=='<':
            if aa*bb-dd>0:
                interval = f"(-\\infty, {solution_ineq})"
            if aa*bb-dd<0:
                interval = f"({solution_ineq},\\infty)"
        if ineq=='<=':
            if aa*bb-dd>0:
                interval = f"(-\\infty, {solution_ineq}]"
            if aa*bb-dd<0:
                interval = f"[{solution_ineq},\\infty)"
        if ineq=='>':
            if aa*bb-dd>0:
                interval = f"({solution_ineq},\\infty)"
            if aa*bb-dd<0:
                interval = f"(-\\infty, {solution_ineq})"        
        if ineq=='>=':
            if aa*bb-dd>0:
                interval = f"[{solution_ineq},\\infty)"
            if aa*bb-dd<0:
                interval = f"(-\\infty, {solution_ineq}]"
```

Comment (siwelwerd) [reply precalculus/exercises/outcomes/EQ/EQ1/generator.sage:56]: a,b,d were for the first task!  The inequality uses aa,bb, and dd.  Probably a sign that these were terrible choices for variable names.

Comment (siwelwerd) [reply precalculus/exercises/outcomes/EQ/EQ1/generator.sage:56]: @StevenClontz please double check me here and commit as appropriate.

**Pull request merged**